bot/exts/filters/token_remover.py

```diff
@@ -167,7 +219,7 @@ def find_token_in_message(cls, msg: Message) -> t.Optional[Token]:
                 return token
 
         # No matching substring
-        return
+        return None
```

Comment:
Why this change?

Reply:
As the function can return a value, I wanted to make it concretely return None while I was here, rather than implicitly return None.